docs/release_notes.rst

```diff
@@ -10,4 +10,4 @@ Qiskit |version| release notes
     `:earliest-version:` should be set to the rc1 release for the current minor release series. For example, the stable/1.1 branch should set it to 1.1.0rc1. If on `main`, set to the prior minor version's rc1, like `1.0.0rc1`.
 
 .. release-notes::
-   :earliest-version: 1.0.0rc1
+   :earliest-version: 1.2.0rc1
```

Comment:
I'm not so sure what this should be, in the 1.1.0 release PR this was set to `1.1.0rc1`, which is why I set it to `1.2.0rc1` -- but better if someone could double check 😄

Reply:
Checking both `stable/1.1.0` and `stable/1.0.0` they each respectively have `1.1.0rc1` and `1.0.0rc1`. As the instructions say here, it should only be `1.0.0rc1` on `main`.